### ISSUE TITLE
Weighted combination of loss functions

### ISSUE THREAD

**Issue by srivathsapv** [voxelmorph/voxelmorph#357]
I was wondering if you guys got a chance to try a combination of different loss functions (MSE, NCC and NMI)? Is there advantages or disadvantages of doing this based on your experiments so far?

@adalca would appreciate your insights into this. Thanks

**Comment by adalca** [comment]
I don't believe we have for registration, could be a simple interesting comparison! I would suspect that NCC will just do most of the work, though.
